src/pkg/retention/controllers/retention.go

```diff
@@ -155,15 +156,15 @@ func (r *RetentionAPI) ListRetentionExecHistory() {
 		r.SendInternalServerError(err)
 		return
 	}
-	query := &q.Query{
-		PageNumber: page,
-		PageSize:   size,
-	}
-	his, err := r.manager.ListHistories(eid, query)
+	tasks, err := r.manager.ListTasks(&q.TaskQuery{
```

Comment:
should put into `retention.APIController`

Reply:
Will change it in the next PR.